content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,393 @@
+3. If you have not generated a Datadog principal for your org, click the **Generate Principal** button.
+4. Copy your Datadog principal and keep it for the next section.
+
+{{< img src="integrations/google_cloud_platform/principal-2.png" alt="Datadog interface, showing the 'Add New GCP Account' flow. The first step, 'Add Datadog Principal to Google,' features a text box where a user can generate a Datadog Principal and copy it to their clipboard. The second step, 'Add Service Account Email,' features a text box that the user can complete in section 4." style="width:70%;">}}
+
+Keep this window open for Section 4.
+5. In the [Google Cloud console][55], under the **Service Accounts** menu, find the service account you created in Section 1.
+6. Go to the **Permissions** tab and click on **Grant Access**.
+
+   {{< img src="integrations/google_cloud_platform/grant-access.png" alt="Google Cloud console interface, showing the Permissions tab under Service Accounts." style="width:70%;">}}
+7. Paste your Datadog principal into the **New principals** text box.
+8. Assign the role of **Service Account Token Creator**.
+9. Click **Save**.
+
+**Note**: If you previously configured access using a shared Datadog principal, you can revoke the permission for that principal after you complete these steps.
+{{% /collapse-content %}}
+
+{{% collapse-content title="3. Add the service account at the organization or folder level" level="h5" %}}
+
+1. In the Google Cloud console, go to the **IAM** page.
+2. Select a folder or organization.
+3. To grant a role to a principal that does not already have other roles on the resource, click **Grant Access**, then enter the email of the service account you created earlier.
+4. Enter the service account's email address.
+5. Assign the following roles:
+- Compute Viewer
+- Monitoring Viewer
+- Cloud Asset Viewer
+6. Click **Save**.
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="4. Complete the integration setup in Datadog" level="h5" %}}
+1. In your Google Cloud console, navigate to the **Service Account** > **Details** tab. On this page, find the email associated with this Google service account. It has the format `<SA_NAME>@<PROJECT-ID>.iam.gserviceaccount.com`.
+2. Copy this email.
+3. Return to the integration configuration tile in Datadog (where you copied your Datadog principal in the previous section).
+4. In the box under **Add Service Account Email**, paste the email you previously copied.
+5. Click on **Verify and Save Account**.
+
+{{% /collapse-content %}}
+
+After finishing these steps, metrics appear in Datadog after approximately **15 minutes**.
+
+## Metric collection
+
+The Google Cloud integration collects all available [Google Cloud metrics][17] from your projects through the Google Cloud Monitoring API.
+
+{{% collapse-content title="Click here for a sample of the Google Cloud metrics Datadog collects" level="h5" %}}
```

Comment:
Is there a way to have this table be centralized since we have the same content on the [README](https://github.com/DataDog/dogweb/blob/prod/integration/google_cloud_platform/README.md?plain=1#L9) md and I can see opportunities for this to get out of sync